gallery/visualtests/nxTag.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { clickTest, focusTest, focusAndHoverTest, hoverTest, simpleTest } = require('./testUtils');
+const { Target } = require('@applitools/eyes-webdriverio');
+
+describe('NxTag', function() {
+  beforeEach(async function() {
+    await browser.url('#/pages/NxTag');
+  });
+
+  describe('Basic NxTag', function() {
+    const selector = '#nx-tag-example .gallery-example-live';
+
+    it('looks right', simpleTest(selector));
+  });
+
+  describe('Show NxTag overflow tooltip', function() {
+    const item2Selector = '#nx-tag-example .nx-tag:nth-child(4)', tooltipSelector = '.nx-tooltip';
```

Comment:
One variable per line please.
```suggestion
    const item2Selector = '#nx-tag-example .nx-tag:nth-child(4)', 
        tooltipSelector = '.nx-tooltip';

```